content/automate/ManageIQ/System/CommonMethods/MiqAe.class/__methods__/weightedupdatestatus.rb

```diff
@@ -139,7 +139,7 @@ def main
                   if @handle.root['ae_state_step'] == 'on_error'
                     task.message = 'Failed'
                     create_cleanup_request(task)
-                  elsif task.get_option('cancel_requested') && task.get_option(:cleanup_request_id).blank?
+                  elsif task.cancelation_status == 'cancel_requested' && task.get_option(:cleanup_request_id).blank?
```

Comment:
Don't hard-code the status strings.  Use the defined methods: 

```ruby
    expose :cancel_requested?
    expose :canceling?
    expose :canceled?
```
https://github.com/ManageIQ/manageiq-automation_engine/blob/master/lib/miq_automation_engine/service_models/miq_ae_service_miq_request_task.rb#L13-L15

Reply:
Sure. Done.